dolphin/processor/config.py

```diff
@@ -288,8 +289,37 @@ def get_kwargs_likelihood(self):
                     prior_param.extend(i)
                     kwargs_likelihood['prior_ps'].append(prior_param)
 
+        if 'lens_light_option' in self.settings and \
+                'prior_lens_light_ellip' in self.settings['lens_light_option']:
+            kwargs_likelihood['custom_logL_addition'] = \
+                self.custom_logL_addition
+
         return kwargs_likelihood
 
+    def custom_logL_addition(self, kwargs_lens=None, kwargs_source=None,
+                             kwargs_lens_light=None, kwargs_ps=None,
+                             kwargs_special=None,
+                             kwargs_extinction=None):
+        """
+        Impose a Gaussian prior on the NFW scale radius R_s based on
+         Gavazzi et al. (2007).
+        """
+
+        pa_light = ellipticity2phi_q(kwargs_lens[0]['e1'],
+                                     kwargs_lens[0]['e2'])[0] * 180 / np.pi
+        pa_mass = \
+            ellipticity2phi_q(kwargs_lens_light[0]['e1'],
+                              kwargs_lens_light[0]['e2'])[0] * 180 / np.pi
+
+        sigma = self.settings['lens_light_option']['prior_lens_light_ellip']
+
+        diff = min(abs(pa_light-pa_mass), 180 - abs(pa_light-pa_mass))
+
+        if diff < sigma:
```

Comment:
What if a user provides negative value for `sigma`? Also, probably call it `max_delta` instead of `sigma` for more clarity, as the prior is uniform here.